constants.go

```diff
@@ -3,9 +3,9 @@ package dotnetcoresdk
 import "regexp"
 
 var Priorities = []interface{}{
+	"global.json",
 	"BP_DOTNET_FRAMEWORK_VERSION",
 	"buildpack.yml",
-	"global.json",
 	regexp.MustCompile(`.*\.(cs)|(fs)|(vb)proj`),
```

Comment:
Is this priorities list in order? Typically, the env variable has the highest precedence, so it can override anything else. Is this changing giving global.json the highest priority?

Reply:
Yes that was intentional, and so far as I can tell none of the other buildpacks actually create a plan entry with `global` as the source so it was unused.

The tricky part is that if I didn't change this order, then there is a scenario where you could not use a specific version of _both_ ASP.NET _and_ SDK - the env var would override the setting specified in `global.json`.

I did consider deprecating this env var, in favour of two separate ones eg `BP_DOTNET_RUNTIME_VERSION` and `BP_DOTNET_SDK_VERSION` but decided it would be better to use the native way with `global.json`.